package.json

```diff
@@ -44,9 +44,9 @@
   },
   "devDependencies": {
     "ava": "*",
-    "eslint": "^1.5.1",
+    "eslint": "^1.6.0",
     "is-plain-obj": "^1.0.0",
-    "temp-write": "^1.1.2"
+    "temp-write": "^2.0.0"
```

Comment:
@sindresorhus There is no difference in `.sync` method between temp-write v1 and v2. https://github.com/sindresorhus/temp-write/commit/27fc22833b75fa89f8dc1b43123ebbce52665b35#diff-168726dbe96b3ce427e7fedce31bb0bc

We can safely bump temp-write to v2.0.0 without changing the code.